source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/camera/tiled_camera.py

```diff
@@ -54,7 +56,7 @@ class TiledCamera(Camera):
     cfg: TiledCameraCfg
     """The configuration parameters."""
 
-    SUPPORTED_TYPES: set[str] = {"rgb", "depth"}
+    SUPPORTED_TYPES: set[str] = {"rgb", "distance_to_camera", "depth"}
     """The set of sensor types that are supported."""
```

Comment:
Do we want to add a NOTE in the docstring that `depth` is an alias for `distance_to_camera`? Knowing full well we will have to change it in the release.